server/gameconnection.py

```diff
@@ -399,14 +444,14 @@ def send_message(self, message):
         game_connection = peer.game_connection
         if not game_connection:
             self._logger.info(
-                "Ignoring ICE message for player without game connection: %s", receiver_id
+                "Ignoring ICE message for player without game connection: %s",
+                receiver_id,
             )
             return
 
-        game_connection.send_message({
-            "command": "IceMsg",
-            "args": [int(self.player.id), ice_msg]
-        })
+        game_connection.send_message(
+            {"command": "IceMsg", "args": [int(self.player.id), ice_msg]}
+        )
```

Comment:
again, don't agree with formatting, before it was - each line is one value, now it's unreadable.